### ISSUE TITLE
Quickly change the code to golang 。right row （pure go）

### ISSUE THREAD

**Issue by icetech233** [google/brotli#1188]
Quickly change the code to golang 。right row （pure go）

**Comment by XenoAmess** [comment]
That is typical go-people: reach out hands, ask for lots, produce nothing, then declare theire doing-nothing application running best performance.
gogin, gogin, doggy gogin.

**Comment by icetech233** [comment]
> That is typical go-people: reach out hands, ask for lots, produce nothing, then declare theire doing-nothing application running best performance. gogin, gogin, doggy gogin.

你在狗叫什么，写你的垃圾java去吧，白嫖kindle书的垃圾

**Comment by icetech233** [comment]
> That is typical go-people: reach out hands, ask for lots, produce nothing, then declare theire doing-nothing application running best performance. gogin, gogin, doggy gogin.

傻逼玩意，用你妈的k8s，llama，你怎么不用java自己写一个

**Comment by XenoAmess** [comment]
> > That is typical go-people: reach out hands, ask for lots, produce nothing, then declare theire doing-nothing application running best performance. gogin, gogin, doggy gogin.
> 
> 你在狗叫什么，写你的垃圾java去吧，白嫖kindle书的垃圾

yes. this is typical go people.wandering everywhere to say Oh You Bad I Go Good. or something.
very very typical go people.

**Comment by icetech233** [comment]
> > > That is typical go-people: reach out hands, ask for lots, produce nothing, then declare theire doing-nothing application running best performance. gogin, gogin, doggy gogin.
> > 
> > 
> > 你在狗叫什么，写你的垃圾java去吧，白嫖kindle书的垃圾
> 
> yes. this is typical go people.wandering everywhere to say Oh You Bad I Go Good. or something. very very typical go people.

臭傻逼

**Comment by XenoAmess** [comment]
> > > > That is typical go-people: reach out hands, ask for lots, produce nothing, then declare theire doing-nothing application running best performance. gogin, gogin, doggy gogin.
> > > 
> > > 
> > > 你在狗叫什么，写你的垃圾java去吧，白嫖kindle书的垃圾
> > 
> > 
> > yes. this is typical go people.wandering everywhere to say Oh You Bad I Go Good. or something. very very typical go people.
> 
> 臭傻逼

This is 'Go Polite'. and go people use this to greet every non-go projects.

**Comment by XenoAmess** [comment]
but whatever, this is a google project, so I have to say maybe they deserve it.
who raise dog get bites.

**Comment by icetech233** [comment]
> but whatever, this is a google project, so I have to say maybe they deserve it.
> who raise dog get bites.

傻逼玩意，不识字
